lms/djangoapps/certificates/generation_handler.py

```diff
@@ -377,11 +377,14 @@ def _get_grade_value(course_grade):
     return ''
 
 
-def _get_course_grade(user, course_key):
+def _get_course_grade(user, course_key, send_grade_signals=True):
```

Comment:
maybe we could change the name to: `send_course_grade_signals` (all signals start with COURSE_GRADE...)

Reply:
Yeap, it will be a piece of cake)